### PR TITLE
Docker compose hobby

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

I copied everything over from here: https://github.com/PostHog/deployment/tree/master/compose

Just renamed to `docker-compose.hobby.yml` and pointed to this repo

## How did you test this code?

*Please describe.*

### CODE REVIEW

File: bin/deploy-hobby [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
```
Comment (guidoiaquinti) [inline bin/deploy-hobby:2]: I know we don't have a CI check for this (yet) but can you run this via https://github.com/koalaman/shellcheck ? There are few warnings that I think we should fix

Comment (yakkomajuri) [reply bin/deploy-hobby:2]: done!

File: bin/deploy-hobby [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+echo ""
+echo "We will need sudo access so the next question is for you to give us superuser access"
+echo "Please enter your sudo password now:"
+sudo echo ""
+echo "Thanks! 🙏"
+echo ""
+echo "Ok! We'll take it from here 🚀"
+
+echo "Making sure any stack that might exist is stopped"
+sudo -E docker-compose -f docker-compose.hobby.yml stop || true
```
Comment (guidoiaquinti) [inline bin/deploy-hobby:34]: why is sudo needed? Because we bind caddy on port 80?

File: bin/deploy-hobby [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+envsubst > Caddyfile <<EOF
+$DOMAIN, :80, :443 {
+reverse_proxy http://web:8000
+}
+EOF
+
+# write entrypoint
+rm -rf compose
+mkdir -p compose
+cat > compose/start <<EOF
```
Comment (guidoiaquinti) [inline bin/deploy-hobby:57]: instead of rewrite another start script, can we reuse something we already have?

File: bin/deploy-hobby [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+./bin/docker-server
+./bin/docker-frontend
+EOF
+chmod +x compose/start
+
+# setup docker
+echo "Setting up Docker"
+sudo apt install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo -E apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
```
Comment (guidoiaquinti) [inline bin/deploy-hobby:70]: is this supposed to work also non non-ubuntu non-linux boxes?

**Comment by yakkomajuri** [comment]
@guidoiaquinti I'd advocate for getting this in and looping @fuziontech in with these concerns later. I'm just cmd+c cmd+v stuff here. 

I'm just being wary of not getting stuck in these side tasks I pick up as I have a tendency to do that.

**Review by guidoiaquinti — APPROVED** [review]
Semi-blind LGTM 🚢

**Comment by yakkomajuri** [comment]
note to self to update docs to reflect this.

**Pull request merged**